ftw/servicenavigation/profiles/default_plone5/registry.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<registry>
+
+    <records prefix="plone.bundles/ftw-servicenavigation-resources" interface='Products.CMFPlone.interfaces.IBundleRegistry'>
+        <value key="enabled">True</value>
+        <value key="depends">plone</value>
+        <value key="stub_js_modules">jquery</value>
+        <value key="compile">False</value>
+        <value key="csscompilation">++resource++servicenavigation/css/servicenavigation-compiled.css</value>
+        <value key="jscompilation">++resource++servicenavigation/js/servicenavigation-compiled.js</value>
+        <value key="merge_with">default</value>
+        <value key="last_compilation">2019-12-07 15:25:00</value>
```

Comment:
Which year are you in?????? ;-)

Reply:
I think it doesn't matter too much, but I can change it if you like...

Reply:
I think it might get used to break the caching of JS files, so we need to remember to update it when we change JS files, no?  So it's got the potential to be confusing if it's not roughly right.

Reply:
👍